distribution/test/itests/test-itests-common/src/test/java/ddf/common/test/PaxExamRuleTest.java

```diff
@@ -187,9 +192,13 @@ public void failingAfterExam() {
         JUnitCore core = new JUnitCore();
         Result result = core.run(FailingAfterExamTest.class);
 
-        assertThat(result.getFailures()).extracting("message")
-                .contains(EXPECTED_AFTER_EXAM_ERROR_MESSAGE, FAILING_TEST_MESSAGE);
-
+        assertThat(result.getFailures()
```

Comment:
Same formatting content. It would be fine if you could move the second boolean term to start on the line after the first so it lined up better but in the absence of that, I think two terms will be easier to read.